src/components/Sheet/README.md

```diff
@@ -0,0 +1,280 @@
+---
+name: Sheet
+category: Overlays
+platforms:
+  - web
+keywords:
+  - sheet
+  - modal
+  - open
+  - title
+  - overlay
+  - drawer
+  - dialog
+  - web
+fullSizeExamples: true
+---
+
+# Sheet
+
+A sheet is a large container that enters from the edge of the screen when triggered by the merchant. It’s used to provide merchants with actions and information contextual to the page. It doesn’t interrupt their flow like a modal.
+
+---
+
+## Required components
+
+The sheet component must be wrapped in the [frame](/components/structure/frame) component.
+
+---
+
+## Use in an embedded application
+
+Use of the sheet component in an embedded application is not currently supported. If this is a feature you would like to see supported by Shopify App Bridge, let us know in the [forums](https://community.shopify.com/c/Shopify-APIs-SDKs/bd-p/shopify-apis-and-technology).
+
+---
+
+## Accessibility
```

Comment:
There's a trick to getting the accessibility content to show up in the right place in the style guide, since it's platform specific. I'll open a PR to update that.